pkg/kubelet/kubelet.go

```diff
@@ -1464,6 +1472,7 @@ func (kl *Kubelet) SyncPods(allPods []api.Pod, podSyncTypes map[types.UID]metric
 		if m, ok := mirrorPods[podFullName]; ok {
 			mirrorPod = &m
 		}
+		glog.Infof("######## SYNCPODS: pod %q, mirrorPod %v", podFullName, mirrorPod)
```

Comment:
Haha it's hard to ignore when looking at the integration test output... I need to have a better system to keep track of my random debugging messages :|